Sources/Imperial/Routing/FederatedServiceRouter.swift

```diff
@@ -58,23 +58,16 @@ public protocol FederatedServiceRouter {
 }
 
 extension FederatedServiceRouter {
-    public func configureRoutes(withAuthURL authURL: String, authenticateCallback: ((Request) throws -> (EventLoopFuture<Void>))?, on router: Routes) throws {
-//        var callbackPath: String = callbackURL
-//        if try NSRegularExpression(pattern: "^https?:\\/\\/", options: []).matches(in: callbackURL, options: [], range: NSMakeRange(0, callbackURL.utf8.count)).count > 0 {
-//            callbackPath = URL(string: callbackPath)?.path ?? callbackURL
-//        }
-//        callbackPath = callbackPath != "/" ? callbackPath : callbackURL
-//
-//        router.get(callbackPath, use: callback)
-//        router.get(authURL) { req -> EventLoopFuture<Response> in
-//            let redirect: Response = req.redirect(to: try self.authURL(req))
-//            guard let authenticateCallback = authenticateCallback else {
-//                return req.eventLoop.makeSucceededFuture(redirect)
-//            }
-//            return try authenticateCallback(req).map(to: Response.self) { _ in
-//                return redirect
-//            }
-//        }
-        fatalError()
+    public func configureRoutes(withAuthURL authURL: String, authenticateCallback: ((Request) throws -> (EventLoopFuture<Void>))?, on router: RoutesBuilder) throws {
+		router.get(callbackURL.pathComponents, use: callback)
```

Comment:
Can you add the (commented out) code that sets the `callbackPath` to HTTPS if needed?

Reply:
These routes depend from the vapor configuration. Is it possible to make certains path to HTTPS if the rest of them are HTTP?